requirements/edx/base.in

```diff
@@ -75,7 +75,7 @@ edx-django-release-util             # Release utils for the edx release pipeline
 edx-django-sites-extensions==2.3.1
 edx-django-utils
 edx-drf-extensions
-edx-enterprise==1.6.9
+edx-enterprise
```

Comment:
@jmbowman - i noticed you had pinned edx-enterprise here - any reason not to unpin?